src/actions/guides/database/querying-deleting.cr

```diff
@@ -556,6 +557,54 @@ class Guides::Database::QueryingDeleting < GuideAction
     task.user!
     ```
 
+    #{permalink(ANCHOR_RELOADING)}
+    ## Reloading
+
+    Reloading a model can be useful when you've loaded a model, but then there is a change to the data.
+
+    ```crystal
+    author = AuthorQuery.find(5)
+
+    # Let's say the Author's profile picture is hidden
+    author.hide_face #=> true
+
+    # If this database value is updated...
+    SaveAuthor.update!(author, hide_face: false)
+
+    # We can reload to get the new value
+    author.reload.hide_face #=> false
+    ```
+
+    When calling the `reload` method on the [model](#{Guides::Database::Models.path}), the original
+    instance is not updated.
+
+    ```crystal
+    # The new value grabbed from the reloaded model
+    author.reload.hide_face #=> false
+
+    # The original value is still in place
+    author.hide_face #=> true
+    ```
+
+    ### Reloading with preloading
+
+    You can also use the `reload` method to preload associations. For example, if you
+    have a post, and want to get the comments preloaded, you can use `reload` with a block. This
+    will help you with performance by avoiding N+1 performance issues.
```

Comment:
I think this line could be removed or reworded since I think the only reason to do this is for performance reasons. So maybe "This will help with performance by avoiding N+1" but maybe it would be better to remove it and link to the preloading docs? ANd put the N+1 stuff there?